packages/react-atlas-default-theme/src/table/reactBsTable.global.css

```diff
@@ -0,0 +1,314 @@
+/*editor error animate*/
+/*@import "~toastr/build/toastr.min.css";*/
+
```

Comment:
We should put the bootstrap license at the top of this file so people know it's bootstrap code and falls under their license.